tests/NHSD.BuyingCatalogue.Ordering.Common.UnitTests/Builders/OrderBuilder.cs

```diff
@@ -26,7 +26,7 @@ public sealed class OrderBuilder
         private bool _catalogueSolutionsViewed;
         private bool _additionalServicesViewed;
         private bool _associatedServicesViewed;
-        private bool? _fundingSourceOnlyGMS;
+        private bool? _fundingSourceOnlyGms;
```

Comment:
Just curious, why did you change the name for this? Is it due to  resharper?

Reply:
Yeah Resharper complains about the capitals. I thought we'd said ages ago in one of the coding standards meetings we wouldn't capitalise acronyms e.g. API

Reply:
Ahhh, my bad. My memory isn't the best 😆

Reply:
😂 That one only stuck with me because I introduced API somewhere & got shot down.